inst/textdata/namelist.tsv

```diff
@@ -494,59 +494,73 @@ caves	nl	grotten	NA
 ep_061	nl	61 Toename overstromingsduur of -frequentie (incl. getijden)	061_overstr_plus
-ep_061_explan	nl	"Een toename resp. afname van het overstromingsregime van waterlopen in zodanige mate dat dit de beschouwde levensgemeenschap buiten het aquatische milieu negatief beïnvloedt. Dit gebeurt door de verandering in mechanische belasting (stroming/sediment), of door de zuurstof/CO2-armoede van overstromingswater.
-Milieudruk 61 is daarmee van toepassing op alle niet-aquatische types, milieudruk 62 op alle oppervlaktewaterafhankelijke niet-aquatische types (niet-aquatisch: betreft de types van de waterhuishoudingsklassen 'droog tot vochtig' en 'tijdelijk tot permanent nat' volgens systeemschema)."	NA
+ep_061_explan	nl	Een toename van het overstromingsregime van waterlopen in zodanige mate dat dit de beschouwde levensgemeenschap buiten het aquatische milieu negatief beïnvloedt. Dit gebeurt door de verandering in mechanische belasting (stroming/sediment), of door de zuurstof/CO2-armoede van overstromingswater. De milieudruk is relevant voor terrestrische types (habitattypes/RBB; 'terrestrisch' betreft de types van de waterhuishoudingsklassen 'droog tot vochtig' en 'tijdelijk tot permanent nat' volgens het systeemschema).	NA
 ep_062	nl	62 Afname overstromingsduur of -frequentie (incl. getijden)	062_overstr_min
+ep_062_explan	nl	"Een afname van het overstromingsregime van waterlopen in zodanige mate dat dit de beschouwde levensgemeenschap buiten het aquatische milieu negatief beïnvloedt. Dit gebeurt door de verandering in mechanische belasting (stroming/sediment), of door de zuurstof/CO2-armoede van overstromingswater.
+Gevoeligheid voor milieudruk 62 definieert de zg. 'overstromingsafhankelijke' types (deze zijn steeds terrestrisch). Het betreft een subset van types (habitattypes/RBB) uit de waterhuishoudingsklasse 'tijdelijk tot permanent nat' volgens het systeemschema."	NA
 ep_063	nl	63 Toename van stroomsnelheid, waterpeil en/of de fluctuatie ervan	063_owpeil_plus
-ep_063_explan	nl	"Een toename resp. afname van de hydraulische of waterpeilregimekarakteristieken ter hoogte van de beschouwde aquatische of semi-terrestrische levensgemeenschap, in zodanige mate dat deze er negatief door wordt beïnvloed.
-Milieudrukken 63 en 64 zijn daarmee van toepassing op alle 'vrijwel steeds overstromingsgestuurde' types uitgezonderd 1310_pol, 1330_hpr, 6120, 6510_hua, 91E0_vn, 91F0, rbbsf en rbbzil."	NA
+ep_063_explan	nl	Een toename van de hydraulische of waterpeilregimekarakteristieken ter hoogte van de beschouwde aquatische levensgemeenschap (oppervlaktewater) of terrestrische levensgemeenschap (bij aanwezigheid van inundatiewater), in zodanige mate dat deze er negatief door wordt beïnvloed.	NA
```

Comment:
Hier vind ik vind de oorspronkelijke tekst beter. Het gaat hier m.i. naast aquatische typen enkel over verlandingsvegetaties die (behoorlijk) rechtstreeks door het oppervlaktewater worden beïnvloed. Inundatiewater beschouw ik al meer als een indirecte invloed en zou ik tot 61 of 62 rekenen.

Reply:
OK om oorspronkelijke bewoording te behouden. Een optie (was voor mij de aanleiding) blijft m.i. wel om 'semi-terrestrisch' te vermijden (maar 'terrestrisch' te gebruiken) aangezien de term 'semi-terrestrisch' niet 1-op-1 met een ander element (drukgevoeligheid, waterhuishoudingsklasse, ...) overeenkomt en daardoor verwarrend/vaag kan overkomen. Niet dat ik dat zo belangrijk vind, een beetje vaagheid kan.

Ivm terminologie oppervlaktewater/inundatiewater in je comment: een kwestie hier is dat de grens in het systeemschema tussen aquatisch en terrestrisch gepaard gaat met de grens tussen 'waterkolom' en 'inundatiewater'.

- Met andere woorden, het compartiment inundatiewater heeft (per definitie volgens systeemschema) een bredere betekenis: alle oppervlaktewater in terrestrisch (= niet-aquatisch) milieu is benoemd als 'inundatiewater'.
- Het betekent wel dat ik 'oppervlaktewater' in mijn voorstel had moeten vervangen door 'waterkolom'.

Reply:
Enige overweging hier is 'semi-terrestrisch' behouden of vervangen door 'terrestrisch', in de oorspronkelijke definitie. Wat denk je @w-jan ?

Reply:
Conclusie: we verkiezen 'semi-terrestrisch'. (Een andere optie was 'verlandingsgemeenschap', maar dit correspondeert niet met alle betrokken types, bv. schorren)

Reply:
Implemented in b6c86b3.